src/components/Settings/SettingsContent.jsx

```diff
@@ -14,38 +14,93 @@ const SettingsContent = ({
   onClaudy,
   isDrawer = false,
```

Comment:
What is the meaning of the isDrawer ? Is there a mobile responsive version ?

Reply:
Yep. But I don't know if this api is still used :/

Reply:
It seems to be used at the root of the Bar for the moment